.devcontainer/contributor/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "Radius CLI - Contributor",
+    "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
+    "features": {
+        "ghcr.io/devcontainers/features/go:1": {
+            "version": "1.21"
+        },
+        "ghcr.io/guiyomh/features/golangci-lint:0": {},
```

Comment:
It looks like the Go feature also includes `golangci-lint`.

Reply:
Cross checked, you are right -> removing the additional feature